plugins/module_utils/client.py

```diff
@@ -48,7 +48,7 @@ def json(self) -> Any:
             try:
                 self._json = json.loads(self.data)
             except ValueError:
-                raise ScaleComputingError(
+                raise ValueError(
```

Comment:
This was the actual problem ... We should really remove all `raise ScaleComputingError` from client and rest_client.